build.gradle.kts

```diff
@@ -65,21 +66,19 @@ dependencyAnalysis {
   }
 }
 
-val testShardNonHibernate by tasks.creating() {
+val testShardNonHibernate: Task by tasks.creating {
   group = "Continuous integration"
   description = "Runs all tests that don't depend on misk-hibernate. " +
     "This target is intended for manually sharding tests to make CI faster."
 }
 
-val testShardHibernate by tasks.creating() {
+val testShardHibernate: Task by tasks.creating {
   group = "Continuous integration"
   description = "Runs all tests that depend on misk-hibernate. " +
     "This target is intended for manually sharding tests to make CI faster."
 }
 
 subprojects {
-  apply(plugin = "org.jetbrains.dokka")
```

Comment:
I think I can remove this since `alias(libs.plugins.dokkaGradlePlugin)` is in the root plugins?